subt_ign/src/GameLogicPlugin.cc

```diff
@@ -829,6 +899,7 @@ void GameLogicPluginPrivate::OnEvent(const ignition::msgs::Pose &_msg)
     }
   }
   this->LogEvent(stream.str());
+  this->PublishRegionEvent("detect", _msg.name(), frameId, state);
```

Comment:
Type should be `artifact_proximity`, `vertical`, `elevation`, `rock_fall`, etc. (contained in extraData) rather than always "detect".

Reply:
Done: e606acbe1c7532603535b556ac26ba069eaafc85